components/list/list-item-generic-layout.js

```diff
@@ -67,6 +67,17 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 				grid-column: content-start / end;
 				grid-row: nested-start / nested-end;
 			}
+			:host(.d2l-dragging-over) ::slotted([slot="nested"]) {
```

Comment:
We need to make sure the nested list is stacked on top of the outer list-item's drop target when dragging over, otherwise we can never activate the nested list items' drop targets.